Jenkinsfile-release-config

```diff
@@ -0,0 +1,44 @@
+pipeline{
+	agent any
+	
+	stages{
+		stage('Confirm configurations'){
+			steps{
+				script{
+          				def userInput = input(
+          				id: 'userInput', message: "Has the Config file credentials been updated for this release?",
+          				parameters: [
+                      					[$class: 'TextParameterDefinition', defaultValue: '', description: '', name: 'response']
+                      				])
+					
+          				if (!userInput.toLowerCase().startsWith("y")) {
+						error("Please update the config file for this release and upload it in Jenkins as a secret file credential. Please have the credential scoped to this folder.")
+					}
+				}
+				script{
+				       	def userInput = input(
+          				id: 'userInput', message: "Have the release-specific Jenkins environmental variables been updated for this release?",
+          				parameters: [
+                      					[$class: 'TextParameterDefinition', defaultValue: '', description: '', name: 'response']
+                      				])
+					
+					if (!userInput.toLowerCase().startsWith("y")) {
+						error("Please update release-specific environmental variables found at 'Jenkins -> Manage Jenkins -> Configure System -> Global Properties'.")
+					}
```

Comment:
I think Jenkins allows the creation of functions.  Extracting out the logic would be useful here since it is very similar logic to the check above for the config file.

Reply:
Yes it allows functions, though looking at it just now I think the amount of boilerplate surrounding this would be adding as much if not more code. If this module expands to add more questions of exactly the same structure I would want to do it, but for simplicity's sake I'd prefer to leave it as is.